Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
+    <header>
+      <h1>Understanding Git</h1>
+      <p>This is all about understanding how git works.</p>
+  </header>
+
+  
+      <article id="what-is-git">
+          <h2>What is Git?</h2>
+          <p>Git is a distributed version control system that helps developers track changes in their code over time. It is widely used to manage project versions and collaborate with other developers efficiently.</p>
+
+          <a href="#">Learn More</a>
+      </article>
+
+      <div class="article2-article3">
+      <div>
+      <article class="section" id="why-developers-need-git">
```

Comment:
the second and third article should each take up about half the viewport -- how might you accomplish this?